docs/notebooks/figanos_docs.ipynb

```diff
@@ -54,12 +54,32 @@
   },
```

Comment:
<p>would move the last sentence to the next markdown cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>